frontend/src/metabase/components/Popover/Popover.module.css

```diff
@@ -15,14 +15,15 @@
   max-width: 500px;
 }
 
-:global(.PopoverBody.PopoverBody--withBackground) {
+.PopoverBody.PopoverBodyWithBackground {
   border: 1px solid #edf2f5;
   box-shadow: 0 4px 10px var(--color-shadow);
   background-color: var(--color-bg-white);
   border-radius: 6px;
   overflow: auto;
 }
 
+/* Global classes from tippy.js */
```

Comment:
We don't migrate global classes for libraries since we don't control those classes.